<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yurClientView</class>
 <widget class="QDialog" name="yurClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Данные организации</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Наименование</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineInn"/>
      </item>
      <item row="2" column="1" alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ИНН</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Юридический адрес</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Индекс</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Город</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineInd"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineCity"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Улица</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Дом</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineStreet"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineBld"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Корпус</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Помещение</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="lineCorp"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineRoom"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineCompany"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Орг.-правовая форма</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineOpf"/>
      </item>
      <item row="2" column="0" alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ОГРН</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignTop">
       <widget class="QLineEdit" name="lineOgrn"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTextEdit" name="textNotes"/>
   </item>
   <item row="0" column="1" rowspan="2" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Контактные данные</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Телефон</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Адрес эл. почты</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineTel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>101</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEmail">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>151</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Сведения о представителе</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineMiddel"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Доверенность или иной документ</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Наименование</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineDoc"/>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Реквизиты</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineReqs"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineNamedel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Должность</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineSurdel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineDel"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Руководитель</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineBsur"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineBname"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineBmid"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="4" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="buttonCancelYur">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="buttonSaveYur">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
